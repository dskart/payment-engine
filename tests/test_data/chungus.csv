type,client,tx,amount
deposit,1,1,1.0
deposit,1,2,1.0
deposit,1,3,1.0
deposit,1,4,1.0
deposit,1,5,1.0
deposit,1,6,1.0
deposit,1,7,1.0
deposit,1,8,1.0
deposit,1,9,1.0
deposit,1,10,1.0
deposit,1,11,1.0
deposit,1,12,1.0
deposit,1,13,1.0
deposit,1,14,1.0
deposit,1,15,1.0
deposit,1,16,1.0
deposit,1,17,1.0
deposit,1,18,1.0
deposit,1,19,1.0
deposit,1,20,1.0
deposit,1,21,1.0
deposit,1,22,1.0
deposit,1,23,1.0
deposit,1,24,1.0
deposit,1,25,1.0
deposit,1,26,1.0
deposit,1,27,1.0
deposit,1,28,1.0
deposit,1,29,1.0
deposit,1,30,1.0
deposit,1,31,1.0
deposit,1,32,1.0
deposit,1,33,1.0
deposit,1,34,1.0
deposit,1,35,1.0
deposit,1,36,1.0
deposit,1,37,1.0
deposit,1,38,1.0
deposit,1,39,1.0
deposit,1,40,1.0
deposit,1,41,1.0
deposit,1,42,1.0
deposit,1,43,1.0
deposit,1,44,1.0
deposit,1,45,1.0
deposit,1,46,1.0
deposit,1,47,1.0
deposit,1,48,1.0
deposit,1,49,1.0
deposit,1,50,1.0
deposit,1,51,1.0
deposit,1,52,1.0
deposit,1,53,1.0
deposit,1,54,1.0
deposit,1,55,1.0
deposit,1,56,1.0
deposit,1,57,1.0
deposit,1,58,1.0
deposit,1,59,1.0
deposit,1,60,1.0
deposit,1,61,1.0
deposit,1,62,1.0
deposit,1,63,1.0
deposit,1,64,1.0
deposit,1,65,1.0
deposit,1,66,1.0
deposit,1,67,1.0
deposit,1,68,1.0
deposit,1,69,1.0
deposit,1,70,1.0
deposit,1,71,1.0
deposit,1,72,1.0
deposit,1,73,1.0
deposit,1,74,1.0
deposit,1,75,1.0
deposit,1,76,1.0
deposit,1,77,1.0
deposit,1,78,1.0
deposit,1,79,1.0
deposit,1,80,1.0
deposit,1,81,1.0
deposit,1,82,1.0
deposit,1,83,1.0
deposit,1,84,1.0
deposit,1,85,1.0
deposit,1,86,1.0
deposit,1,87,1.0
deposit,1,88,1.0
deposit,1,89,1.0
deposit,1,90,1.0
deposit,1,91,1.0
deposit,1,92,1.0
deposit,1,93,1.0
deposit,1,94,1.0
deposit,1,95,1.0
deposit,1,96,1.0
deposit,1,97,1.0
deposit,1,98,1.0
deposit,1,99,1.0
deposit,1,100,1.0
deposit,1,101,1.0
deposit,1,102,1.0
deposit,1,103,1.0
deposit,1,104,1.0
deposit,1,105,1.0
deposit,1,106,1.0
deposit,1,107,1.0
deposit,1,108,1.0
deposit,1,109,1.0
deposit,1,110,1.0
deposit,1,111,1.0
deposit,1,112,1.0
deposit,1,113,1.0
deposit,1,114,1.0
deposit,1,115,1.0
deposit,1,116,1.0
deposit,1,117,1.0
deposit,1,118,1.0
deposit,1,119,1.0
deposit,1,120,1.0
deposit,1,121,1.0
deposit,1,122,1.0
deposit,1,123,1.0
deposit,1,124,1.0
deposit,1,125,1.0
deposit,1,126,1.0
deposit,1,127,1.0
deposit,1,128,1.0
deposit,1,129,1.0
deposit,1,130,1.0
deposit,1,131,1.0
deposit,1,132,1.0
deposit,1,133,1.0
deposit,1,134,1.0
deposit,1,135,1.0
deposit,1,136,1.0
deposit,1,137,1.0
deposit,1,138,1.0
deposit,1,139,1.0
deposit,1,140,1.0
deposit,1,141,1.0
deposit,1,142,1.0
deposit,1,143,1.0
deposit,1,144,1.0
deposit,1,145,1.0
deposit,1,146,1.0
deposit,1,147,1.0
deposit,1,148,1.0
deposit,1,149,1.0
deposit,1,150,1.0
deposit,1,151,1.0
deposit,1,152,1.0
deposit,1,153,1.0
deposit,1,154,1.0
deposit,1,155,1.0
deposit,1,156,1.0
deposit,1,157,1.0
deposit,1,158,1.0
deposit,1,159,1.0
deposit,1,160,1.0
deposit,1,161,1.0
deposit,1,162,1.0
deposit,1,163,1.0
deposit,1,164,1.0
deposit,1,165,1.0
deposit,1,166,1.0
deposit,1,167,1.0
deposit,1,168,1.0
deposit,1,169,1.0
deposit,1,170,1.0
deposit,1,171,1.0
deposit,1,172,1.0
deposit,1,173,1.0
deposit,1,174,1.0
deposit,1,175,1.0
deposit,1,176,1.0
deposit,1,177,1.0
deposit,1,178,1.0
deposit,1,179,1.0
deposit,1,180,1.0
deposit,1,181,1.0
deposit,1,182,1.0
deposit,1,183,1.0
deposit,1,184,1.0
deposit,1,185,1.0
deposit,1,186,1.0
deposit,1,187,1.0
deposit,1,188,1.0
deposit,1,189,1.0
deposit,1,190,1.0
deposit,1,191,1.0
deposit,1,192,1.0
deposit,1,193,1.0
deposit,1,194,1.0
deposit,1,195,1.0
deposit,1,196,1.0
deposit,1,197,1.0
deposit,1,198,1.0
deposit,1,199,1.0
deposit,1,200,1.0
deposit,1,201,1.0
deposit,1,202,1.0
deposit,1,203,1.0
deposit,1,204,1.0
deposit,1,205,1.0
deposit,1,206,1.0
deposit,1,207,1.0
deposit,1,208,1.0
deposit,1,209,1.0
deposit,1,210,1.0
deposit,1,211,1.0
deposit,1,212,1.0
deposit,1,213,1.0
deposit,1,214,1.0
deposit,1,215,1.0
deposit,1,216,1.0
deposit,1,217,1.0
deposit,1,218,1.0
deposit,1,219,1.0
deposit,1,220,1.0
deposit,1,221,1.0
deposit,1,222,1.0
deposit,1,223,1.0
deposit,1,224,1.0
deposit,1,225,1.0
deposit,1,226,1.0
deposit,1,227,1.0
deposit,1,228,1.0
deposit,1,229,1.0
deposit,1,230,1.0
deposit,1,231,1.0
deposit,1,232,1.0
deposit,1,233,1.0
deposit,1,234,1.0
deposit,1,235,1.0
deposit,1,236,1.0
deposit,1,237,1.0
deposit,1,238,1.0
deposit,1,239,1.0
deposit,1,240,1.0
deposit,1,241,1.0
deposit,1,242,1.0
deposit,1,243,1.0
deposit,1,244,1.0
deposit,1,245,1.0
deposit,1,246,1.0
deposit,1,247,1.0
deposit,1,248,1.0
deposit,1,249,1.0
deposit,1,250,1.0
deposit,1,251,1.0
deposit,1,252,1.0
deposit,1,253,1.0
deposit,1,254,1.0
deposit,1,255,1.0
deposit,1,256,1.0
deposit,1,257,1.0
deposit,1,258,1.0
deposit,1,259,1.0
deposit,1,260,1.0
deposit,1,261,1.0
deposit,1,262,1.0
deposit,1,263,1.0
deposit,1,264,1.0
deposit,1,265,1.0
deposit,1,266,1.0
deposit,1,267,1.0
deposit,1,268,1.0
deposit,1,269,1.0
deposit,1,270,1.0
deposit,1,271,1.0
deposit,1,272,1.0
deposit,1,273,1.0
deposit,1,274,1.0
deposit,1,275,1.0
deposit,1,276,1.0
deposit,1,277,1.0
deposit,1,278,1.0
deposit,1,279,1.0
deposit,1,280,1.0
deposit,1,281,1.0
deposit,1,282,1.0
deposit,1,283,1.0
deposit,1,284,1.0
deposit,1,285,1.0
deposit,1,286,1.0
deposit,1,287,1.0
deposit,1,288,1.0
deposit,1,289,1.0
deposit,1,290,1.0
deposit,1,291,1.0
deposit,1,292,1.0
deposit,1,293,1.0
deposit,1,294,1.0
deposit,1,295,1.0
deposit,1,296,1.0
deposit,1,297,1.0
deposit,1,298,1.0
deposit,1,299,1.0
deposit,1,300,1.0
deposit,1,301,1.0
deposit,1,302,1.0
deposit,1,303,1.0
deposit,1,304,1.0
deposit,1,305,1.0
deposit,1,306,1.0
deposit,1,307,1.0
deposit,1,308,1.0
deposit,1,309,1.0
deposit,1,310,1.0
deposit,1,311,1.0
deposit,1,312,1.0
deposit,1,313,1.0
deposit,1,314,1.0
deposit,1,315,1.0
deposit,1,316,1.0
deposit,1,317,1.0
deposit,1,318,1.0
deposit,1,319,1.0
deposit,1,320,1.0
deposit,1,321,1.0
deposit,1,322,1.0
deposit,1,323,1.0
deposit,1,324,1.0
deposit,1,325,1.0
deposit,1,326,1.0
deposit,1,327,1.0
deposit,1,328,1.0
deposit,1,329,1.0
deposit,1,330,1.0
deposit,1,331,1.0
deposit,1,332,1.0
deposit,1,333,1.0
deposit,1,334,1.0
deposit,1,335,1.0
deposit,1,336,1.0
deposit,1,337,1.0
deposit,1,338,1.0
deposit,1,339,1.0
deposit,1,340,1.0
deposit,1,341,1.0
deposit,1,342,1.0
deposit,1,343,1.0
deposit,1,344,1.0
deposit,1,345,1.0
deposit,1,346,1.0
deposit,1,347,1.0
deposit,1,348,1.0
deposit,1,349,1.0
deposit,1,350,1.0
deposit,1,351,1.0
deposit,1,352,1.0
deposit,1,353,1.0
deposit,1,354,1.0
deposit,1,355,1.0
deposit,1,356,1.0
deposit,1,357,1.0
deposit,1,358,1.0
deposit,1,359,1.0
deposit,1,360,1.0
deposit,1,361,1.0
deposit,1,362,1.0
deposit,1,363,1.0
deposit,1,364,1.0
deposit,1,365,1.0
deposit,1,366,1.0
deposit,1,367,1.0
deposit,1,368,1.0
deposit,1,369,1.0
deposit,1,370,1.0
deposit,1,371,1.0
deposit,1,372,1.0
deposit,1,373,1.0
deposit,1,374,1.0
deposit,1,375,1.0
deposit,1,376,1.0
deposit,1,377,1.0
deposit,1,378,1.0
deposit,1,379,1.0
deposit,1,380,1.0
deposit,1,381,1.0
deposit,1,382,1.0
deposit,1,383,1.0
deposit,1,384,1.0
deposit,1,385,1.0
deposit,1,386,1.0
deposit,1,387,1.0
deposit,1,388,1.0
deposit,1,389,1.0
deposit,1,390,1.0
deposit,1,391,1.0
deposit,1,392,1.0
deposit,1,393,1.0
deposit,1,394,1.0
deposit,1,395,1.0
deposit,1,396,1.0
deposit,1,397,1.0
deposit,1,398,1.0
deposit,1,399,1.0
deposit,1,400,1.0
deposit,1,401,1.0
deposit,1,402,1.0
deposit,1,403,1.0
deposit,1,404,1.0
deposit,1,405,1.0
deposit,1,406,1.0
deposit,1,407,1.0
deposit,1,408,1.0
deposit,1,409,1.0
deposit,1,410,1.0
deposit,1,411,1.0
deposit,1,412,1.0
deposit,1,413,1.0
deposit,1,414,1.0
deposit,1,415,1.0
deposit,1,416,1.0
deposit,1,417,1.0
deposit,1,418,1.0
deposit,1,419,1.0
deposit,1,420,1.0
deposit,1,421,1.0
deposit,1,422,1.0
deposit,1,423,1.0
deposit,1,424,1.0
deposit,1,425,1.0
deposit,1,426,1.0
deposit,1,427,1.0
deposit,1,428,1.0
deposit,1,429,1.0
deposit,1,430,1.0
deposit,1,431,1.0
deposit,1,432,1.0
deposit,1,433,1.0
deposit,1,434,1.0
deposit,1,435,1.0
deposit,1,436,1.0
deposit,1,437,1.0
deposit,1,438,1.0
deposit,1,439,1.0
deposit,1,440,1.0
deposit,1,441,1.0
deposit,1,442,1.0
deposit,1,443,1.0
deposit,1,444,1.0
deposit,1,445,1.0
deposit,1,446,1.0
deposit,1,447,1.0
deposit,1,448,1.0
deposit,1,449,1.0
deposit,1,450,1.0
deposit,1,451,1.0
deposit,1,452,1.0
deposit,1,453,1.0
deposit,1,454,1.0
deposit,1,455,1.0
deposit,1,456,1.0
deposit,1,457,1.0
deposit,1,458,1.0
deposit,1,459,1.0
deposit,1,460,1.0
deposit,1,461,1.0
deposit,1,462,1.0
deposit,1,463,1.0
deposit,1,464,1.0
deposit,1,465,1.0
deposit,1,466,1.0
deposit,1,467,1.0
deposit,1,468,1.0
deposit,1,469,1.0
deposit,1,470,1.0
deposit,1,471,1.0
deposit,1,472,1.0
deposit,1,473,1.0
deposit,1,474,1.0
deposit,1,475,1.0
deposit,1,476,1.0
deposit,1,477,1.0
deposit,1,478,1.0
deposit,1,479,1.0
deposit,1,480,1.0
deposit,1,481,1.0
deposit,1,482,1.0
deposit,1,483,1.0
deposit,1,484,1.0
deposit,1,485,1.0
deposit,1,486,1.0
deposit,1,487,1.0
deposit,1,488,1.0
deposit,1,489,1.0
deposit,1,490,1.0
deposit,1,491,1.0
deposit,1,492,1.0
deposit,1,493,1.0
deposit,1,494,1.0
deposit,1,495,1.0
deposit,1,496,1.0
deposit,1,497,1.0
deposit,1,498,1.0
deposit,1,499,1.0
deposit,1,500,1.0
deposit,1,501,1.0
deposit,1,502,1.0
deposit,1,503,1.0
deposit,1,504,1.0
deposit,1,505,1.0
deposit,1,506,1.0
deposit,1,507,1.0
deposit,1,508,1.0
deposit,1,509,1.0
deposit,1,510,1.0
deposit,1,511,1.0
deposit,1,512,1.0
deposit,1,513,1.0
deposit,1,514,1.0
deposit,1,515,1.0
deposit,1,516,1.0
deposit,1,517,1.0
deposit,1,518,1.0
deposit,1,519,1.0
deposit,1,520,1.0
deposit,1,521,1.0
deposit,1,522,1.0
deposit,1,523,1.0
deposit,1,524,1.0
deposit,1,525,1.0
deposit,1,526,1.0
deposit,1,527,1.0
deposit,1,528,1.0
deposit,1,529,1.0
deposit,1,530,1.0
deposit,1,531,1.0
deposit,1,532,1.0
deposit,1,533,1.0
deposit,1,534,1.0
deposit,1,535,1.0
deposit,1,536,1.0
deposit,1,537,1.0
deposit,1,538,1.0
deposit,1,539,1.0
deposit,1,540,1.0
deposit,1,541,1.0
deposit,1,542,1.0
deposit,1,543,1.0
deposit,1,544,1.0
deposit,1,545,1.0
deposit,1,546,1.0
deposit,1,547,1.0
deposit,1,548,1.0
deposit,1,549,1.0
deposit,1,550,1.0
deposit,1,551,1.0
deposit,1,552,1.0
deposit,1,553,1.0
deposit,1,554,1.0
deposit,1,555,1.0
deposit,1,556,1.0
deposit,1,557,1.0
deposit,1,558,1.0
deposit,1,559,1.0
deposit,1,560,1.0
deposit,1,561,1.0
deposit,1,562,1.0
deposit,1,563,1.0
deposit,1,564,1.0
deposit,1,565,1.0
deposit,1,566,1.0
deposit,1,567,1.0
deposit,1,568,1.0
deposit,1,569,1.0
deposit,1,570,1.0
deposit,1,571,1.0
deposit,1,572,1.0
deposit,1,573,1.0
deposit,1,574,1.0
deposit,1,575,1.0
deposit,1,576,1.0
deposit,1,577,1.0
deposit,1,578,1.0
deposit,1,579,1.0
deposit,1,580,1.0
deposit,1,581,1.0
deposit,1,582,1.0
deposit,1,583,1.0
deposit,1,584,1.0
deposit,1,585,1.0
deposit,1,586,1.0
deposit,1,587,1.0
deposit,1,588,1.0
deposit,1,589,1.0
deposit,1,590,1.0
deposit,1,591,1.0
deposit,1,592,1.0
deposit,1,593,1.0
deposit,1,594,1.0
deposit,1,595,1.0
deposit,1,596,1.0
deposit,1,597,1.0
deposit,1,598,1.0
deposit,1,599,1.0
deposit,1,600,1.0
deposit,1,601,1.0
deposit,1,602,1.0
deposit,1,603,1.0
deposit,1,604,1.0
deposit,1,605,1.0
deposit,1,606,1.0
deposit,1,607,1.0
deposit,1,608,1.0
deposit,1,609,1.0
deposit,1,610,1.0
deposit,1,611,1.0
deposit,1,612,1.0
deposit,1,613,1.0
deposit,1,614,1.0
deposit,1,615,1.0
deposit,1,616,1.0
deposit,1,617,1.0
deposit,1,618,1.0
deposit,1,619,1.0
deposit,1,620,1.0
deposit,1,621,1.0
deposit,1,622,1.0
deposit,1,623,1.0
deposit,1,624,1.0
deposit,1,625,1.0
deposit,1,626,1.0
deposit,1,627,1.0
deposit,1,628,1.0
deposit,1,629,1.0
deposit,1,630,1.0
deposit,1,631,1.0
deposit,1,632,1.0
deposit,1,633,1.0
deposit,1,634,1.0
deposit,1,635,1.0
deposit,1,636,1.0
deposit,1,637,1.0
deposit,1,638,1.0
deposit,1,639,1.0
deposit,1,640,1.0
deposit,1,641,1.0
deposit,1,642,1.0
deposit,1,643,1.0
deposit,1,644,1.0
deposit,1,645,1.0
deposit,1,646,1.0
deposit,1,647,1.0
deposit,1,648,1.0
deposit,1,649,1.0
deposit,1,650,1.0
deposit,1,651,1.0
deposit,1,652,1.0
deposit,1,653,1.0
deposit,1,654,1.0
deposit,1,655,1.0
deposit,1,656,1.0
deposit,1,657,1.0
deposit,1,658,1.0
deposit,1,659,1.0
deposit,1,660,1.0
deposit,1,661,1.0
deposit,1,662,1.0
deposit,1,663,1.0
deposit,1,664,1.0
deposit,1,665,1.0
deposit,1,666,1.0
deposit,1,667,1.0
deposit,1,668,1.0
deposit,1,669,1.0
deposit,1,670,1.0
deposit,1,671,1.0
deposit,1,672,1.0
deposit,1,673,1.0
deposit,1,674,1.0
deposit,1,675,1.0
deposit,1,676,1.0
deposit,1,677,1.0
deposit,1,678,1.0
deposit,1,679,1.0
deposit,1,680,1.0
deposit,1,681,1.0
deposit,1,682,1.0
deposit,1,683,1.0
deposit,1,684,1.0
deposit,1,685,1.0
deposit,1,686,1.0
deposit,1,687,1.0
deposit,1,688,1.0
deposit,1,689,1.0
deposit,1,690,1.0
deposit,1,691,1.0
deposit,1,692,1.0
deposit,1,693,1.0
deposit,1,694,1.0
deposit,1,695,1.0
deposit,1,696,1.0
deposit,1,697,1.0
deposit,1,698,1.0
deposit,1,699,1.0
deposit,1,700,1.0
deposit,1,701,1.0
deposit,1,702,1.0
deposit,1,703,1.0
deposit,1,704,1.0
deposit,1,705,1.0
deposit,1,706,1.0
deposit,1,707,1.0
deposit,1,708,1.0
deposit,1,709,1.0
deposit,1,710,1.0
deposit,1,711,1.0
deposit,1,712,1.0
deposit,1,713,1.0
deposit,1,714,1.0
deposit,1,715,1.0
deposit,1,716,1.0
deposit,1,717,1.0
deposit,1,718,1.0
deposit,1,719,1.0
deposit,1,720,1.0
deposit,1,721,1.0
deposit,1,722,1.0
deposit,1,723,1.0
deposit,1,724,1.0
deposit,1,725,1.0
deposit,1,726,1.0
deposit,1,727,1.0
deposit,1,728,1.0
deposit,1,729,1.0
deposit,1,730,1.0
deposit,1,731,1.0
deposit,1,732,1.0
deposit,1,733,1.0
deposit,1,734,1.0
deposit,1,735,1.0
deposit,1,736,1.0
deposit,1,737,1.0
deposit,1,738,1.0
deposit,1,739,1.0
deposit,1,740,1.0
deposit,1,741,1.0
deposit,1,742,1.0
deposit,1,743,1.0
deposit,1,744,1.0
deposit,1,745,1.0
deposit,1,746,1.0
deposit,1,747,1.0
deposit,1,748,1.0
deposit,1,749,1.0
deposit,1,750,1.0
deposit,1,751,1.0
deposit,1,752,1.0
deposit,1,753,1.0
deposit,1,754,1.0
deposit,1,755,1.0
deposit,1,756,1.0
deposit,1,757,1.0
deposit,1,758,1.0
deposit,1,759,1.0
deposit,1,760,1.0
deposit,1,761,1.0
deposit,1,762,1.0
deposit,1,763,1.0
deposit,1,764,1.0
deposit,1,765,1.0
deposit,1,766,1.0
deposit,1,767,1.0
deposit,1,768,1.0
deposit,1,769,1.0
deposit,1,770,1.0
deposit,1,771,1.0
deposit,1,772,1.0
deposit,1,773,1.0
deposit,1,774,1.0
deposit,1,775,1.0
deposit,1,776,1.0
deposit,1,777,1.0
deposit,1,778,1.0
deposit,1,779,1.0
deposit,1,780,1.0
deposit,1,781,1.0
deposit,1,782,1.0
deposit,1,783,1.0
deposit,1,784,1.0
deposit,1,785,1.0
deposit,1,786,1.0
deposit,1,787,1.0
deposit,1,788,1.0
deposit,1,789,1.0
deposit,1,790,1.0
deposit,1,791,1.0
deposit,1,792,1.0
deposit,1,793,1.0
deposit,1,794,1.0
deposit,1,795,1.0
deposit,1,796,1.0
deposit,1,797,1.0
deposit,1,798,1.0
deposit,1,799,1.0
deposit,1,800,1.0
deposit,1,801,1.0
deposit,1,802,1.0
deposit,1,803,1.0
deposit,1,804,1.0
deposit,1,805,1.0
deposit,1,806,1.0
deposit,1,807,1.0
deposit,1,808,1.0
deposit,1,809,1.0
deposit,1,810,1.0
deposit,1,811,1.0
deposit,1,812,1.0
deposit,1,813,1.0
deposit,1,814,1.0
deposit,1,815,1.0
deposit,1,816,1.0
deposit,1,817,1.0
deposit,1,818,1.0
deposit,1,819,1.0
deposit,1,820,1.0
deposit,1,821,1.0
deposit,1,822,1.0
deposit,1,823,1.0
deposit,1,824,1.0
deposit,1,825,1.0
deposit,1,826,1.0
deposit,1,827,1.0
deposit,1,828,1.0
deposit,1,829,1.0
deposit,1,830,1.0
deposit,1,831,1.0
deposit,1,832,1.0
deposit,1,833,1.0
deposit,1,834,1.0
deposit,1,835,1.0
deposit,1,836,1.0
deposit,1,837,1.0
deposit,1,838,1.0
deposit,1,839,1.0
deposit,1,840,1.0
deposit,1,841,1.0
deposit,1,842,1.0
deposit,1,843,1.0
deposit,1,844,1.0
deposit,1,845,1.0
deposit,1,846,1.0
deposit,1,847,1.0
deposit,1,848,1.0
deposit,1,849,1.0
deposit,1,850,1.0
deposit,1,851,1.0
deposit,1,852,1.0
deposit,1,853,1.0
deposit,1,854,1.0
deposit,1,855,1.0
deposit,1,856,1.0
deposit,1,857,1.0
deposit,1,858,1.0
deposit,1,859,1.0
deposit,1,860,1.0
deposit,1,861,1.0
deposit,1,862,1.0
deposit,1,863,1.0
deposit,1,864,1.0
deposit,1,865,1.0
deposit,1,866,1.0
deposit,1,867,1.0
deposit,1,868,1.0
deposit,1,869,1.0
deposit,1,870,1.0
deposit,1,871,1.0
deposit,1,872,1.0
deposit,1,873,1.0
deposit,1,874,1.0
deposit,1,875,1.0
deposit,1,876,1.0
deposit,1,877,1.0
deposit,1,878,1.0
deposit,1,879,1.0
deposit,1,880,1.0
deposit,1,881,1.0
deposit,1,882,1.0
deposit,1,883,1.0
deposit,1,884,1.0
deposit,1,885,1.0
deposit,1,886,1.0
deposit,1,887,1.0
deposit,1,888,1.0
deposit,1,889,1.0
deposit,1,890,1.0
deposit,1,891,1.0
deposit,1,892,1.0
deposit,1,893,1.0
deposit,1,894,1.0
deposit,1,895,1.0
deposit,1,896,1.0
deposit,1,897,1.0
deposit,1,898,1.0
deposit,1,899,1.0
deposit,1,900,1.0
deposit,1,901,1.0
deposit,1,902,1.0
deposit,1,903,1.0
deposit,1,904,1.0
deposit,1,905,1.0
deposit,1,906,1.0
deposit,1,907,1.0
deposit,1,908,1.0
deposit,1,909,1.0
deposit,1,910,1.0
deposit,1,911,1.0
deposit,1,912,1.0
deposit,1,913,1.0
deposit,1,914,1.0
deposit,1,915,1.0
deposit,1,916,1.0
deposit,1,917,1.0
deposit,1,918,1.0
deposit,1,919,1.0
deposit,1,920,1.0
deposit,1,921,1.0
deposit,1,922,1.0
deposit,1,923,1.0
deposit,1,924,1.0
deposit,1,925,1.0
deposit,1,926,1.0
deposit,1,927,1.0
deposit,1,928,1.0
deposit,1,929,1.0
deposit,1,930,1.0
deposit,1,931,1.0
deposit,1,932,1.0
deposit,1,933,1.0
deposit,1,934,1.0
deposit,1,935,1.0
deposit,1,936,1.0
deposit,1,937,1.0
deposit,1,938,1.0
deposit,1,939,1.0
deposit,1,940,1.0
deposit,1,941,1.0
deposit,1,942,1.0
deposit,1,943,1.0
deposit,1,944,1.0
deposit,1,945,1.0
deposit,1,946,1.0
deposit,1,947,1.0
deposit,1,948,1.0
deposit,1,949,1.0
deposit,1,950,1.0
deposit,1,951,1.0
deposit,1,952,1.0
deposit,1,953,1.0
deposit,1,954,1.0
deposit,1,955,1.0
deposit,1,956,1.0
deposit,1,957,1.0
deposit,1,958,1.0
deposit,1,959,1.0
deposit,1,960,1.0
deposit,1,961,1.0
deposit,1,962,1.0
deposit,1,963,1.0
deposit,1,964,1.0
deposit,1,965,1.0
deposit,1,966,1.0
deposit,1,967,1.0
deposit,1,968,1.0
deposit,1,969,1.0
deposit,1,970,1.0
deposit,1,971,1.0
deposit,1,972,1.0
deposit,1,973,1.0
deposit,1,974,1.0
deposit,1,975,1.0
deposit,1,976,1.0
deposit,1,977,1.0
deposit,1,978,1.0
deposit,1,979,1.0
deposit,1,980,1.0
deposit,1,981,1.0
deposit,1,982,1.0
deposit,1,983,1.0
deposit,1,984,1.0
deposit,1,985,1.0
deposit,1,986,1.0
deposit,1,987,1.0
deposit,1,988,1.0
deposit,1,989,1.0
deposit,1,990,1.0
deposit,1,991,1.0
deposit,1,992,1.0
deposit,1,993,1.0
deposit,1,994,1.0
deposit,1,995,1.0
deposit,1,996,1.0
deposit,1,997,1.0
deposit,1,998,1.0
deposit,1,999,1.0
deposit,1,1000,1.0
