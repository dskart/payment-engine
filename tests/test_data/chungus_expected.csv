client,available,held,total,locked
1,1000.0,0.0,1000.0,false
